src/bindings/python/tests/conftest.py

```diff
@@ -89,7 +90,11 @@ def plugins_path():
     base_path = os.path.dirname(__file__)
     plugins_xml = os.path.join(base_path, "test_utils", "utils", "plugins.xml")
```

Comment:
can we parametrize this function `plugins_path` with device name, so the function can dynamically generate `plugins.xml` files with proper plugin file name?
In this case, we can even run the tests on GPU, GNA and other devices.

Reply:
have rewritten `plugins_path`